MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -3084,7 +3084,7 @@ protected string GetHardwareAcceleratedVideoDecoder(EncodingJobInfo state, Encod
                 }
             }
 
-            var whichCodec = videoStream.Codec.ToLowerInvariant();
+            var whichCodec = videoStream.Codec?.ToLowerInvariant();
```

Comment:
Sometimes the videoStream doesn't have a codec- this was throwing nullref exceptions.